### DISCUSSION TITLE
Making Map a default view -  Using Argdown in Markdown with a Web Component

### DISCUSSION THREAD

**Discussion by antonostrovsky** [christianvoigt/argdown#254] [Q&A]
Hello Christian,

Is there a way to make the "Map" view the default HTML view when exporting a web-component?
At present, in my VSCode preview I see "Source" as the default view.

Thank you!

**Comment by christianvoigt** [comment]
Hi Anton, 
for the Markdown preview (and the `argdown markdown` export): simply use an `argdown-map` code block instead of an `argdown` code block:

```markdown
    ```argdown-map
   your argdown code  ...
    ```
```

For the web-component export: here is [how you set the initial view in the frontmatter](https://argdown.org/guide/embedding-your-maps-in-a-webpage.html#set-the-initial-view).

**Comment by christianvoigt** [comment]
But the map view *is* the default view for the web-component export. So you should not have to change anything.

**Comment by antonostrovsky** [comment]
Thank you for the explanation!
